### PR TITLE
chore: enable webpack-dev-server@next on CI

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
chore, CI

**Did you add tests for your changes?**
Already present
**If relevant, did you update the documentation?**
NA
**Summary**
enable webpack-dev-server@next on CI.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Comment by snitin315** [comment]
It seems `webpack-dev-server@next` is not fully compatible. WIP

**Comment by alexander-akait** [comment]
It is fully compatibility. something wrong on our side again

**Comment by snitin315** [comment]
yep, snapshots were for v3. It should be good now.

File: setupTest.js [hunk 1]
```diff
@@ -1 +1 @@
-jest.setTimeout(240000);
+jest.setTimeout(300000);
```
Comment (snitin315) [inline setupTest.js:1]:
```suggestion
jest.setTimeout(420000);
```

File: setupTest.js [hunk 2]
```diff
@@ -1 +1 @@
-jest.setTimeout(240000);
+jest.setTimeout(420000);
```
Comment (snitin315) [inline setupTest.js:1]:
```suggestion
jest.setTimeout(600000);
```

**Comment by alexander-akait** [comment]
Please try to debug locally, there is no problem with timeout

**Comment by snitin315** [comment]
Failing only on MacOS with `async call timeout`. I will take another look tomorrow.

**Comment by snitin315** [comment]
Here something is wrong with actions. It's showing past checks as `Expected`. I have tried closing and reopening the PR. but still the same. I am trying once by recreating PR.

**Pull request closed without merging**